docs/pages/app.md

```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
 Open the following link to display a _Hello World_ message within an open document. Provide any URL encoded string with the `msg` query parameter.
 
 [`sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World`](sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World)
+
+## Add a Remote Library <a name="addremotelibrary"></a>
+
+You can add a Remote Library to Sketch by providing a url to an appcast file.
```

Comment:
I'd make it more explicit that it's about libraries hosted somewhere using RSS. Also for consistency with the other entries, please make sure to use identical formatting of variables and parameter names, plus put the main URL scheme command straight below the summary paragraph. (don't accept suggestion verbatim, indention of snippet isn't correct but required for the suggestion not to break).

```suggestion
Add a Library hosted on any server and shared using RSS by specifying the URL to the appcast.

    ```
    sketch://add-library?url=APPCAST_RSS_URL
    ```

To find out more about sharing your own libraries, see:

- [Sharing a Library](https://www.sketch.com/docs/libraries/#sharing-libraries) Sketch documentation
- [Appcast documentation](https://sparkle-project.org/documentation/)
```

Reply:
Changed.